utils/datasets.py

```diff
@@ -5,7 +5,7 @@
 
 class ImageDataset:
     def __init__(self, dataset = 'cifar_10', batch_sz=256, tfms=None, 
-                resize=None, num_workers=2):
+                resize=None, num_workers=2, drop_last=False):
```

Comment:
scripts/create_fid_stats.py uses the same class to load data and in that case we can't drop batches, hence created a new argument